DISCREPANT METHOD: s3_complete_multipart_upload --- ARGS: []
--Status code mismatch--
CLOUD: FAILURE -- HTTP Status code: 404 -- Error Code: NoSuchUpload -- Error Message: The specified upload does not exist. The upload ID may be invalid, or the upload may have been aborted or completed.
EMULATOR: FAILURE -- HTTP Status code: 500 -- Error Code: InternalError -- Error Message: exception while calling s3.CompleteMultipartUpload: ''



count: 1   ------------------------------------------------------------------------------------------------------------------------------------------------------------------------


DISCREPANT METHOD: s3_delete_bucket_inventory_configuration --- ARGS: []
--Behavior mismatch--
CLOUD: FAILURE -- Response: An error occurred (NoSuchConfiguration) when calling the DeleteBucketInventoryConfiguration operation: The specified configuration does not exist. 
EMULATOR: SUCCESS -- Response: 



count: 2   ------------------------------------------------------------------------------------------------------------------------------------------------------------------------


DISCREPANT METHOD: s3_get_bucket_analytics_configuration --- ARGS: []
--Behavior mismatch--
CLOUD: FAILURE -- Response: An error occurred (NoSuchConfiguration) when calling the GetBucketAnalyticsConfiguration operation: The specified configuration does not exist. 
EMULATOR: SUCCESS -- Response: 



count: 3   ------------------------------------------------------------------------------------------------------------------------------------------------------------------------


DISCREPANT METHOD: s3_get_bucket_intelligent_tiering_configuration --- ARGS: []
--Behavior mismatch--
CLOUD: FAILURE -- Response: An error occurred (NoSuchConfiguration) when calling the GetBucketIntelligentTieringConfiguration operation: The specified configuration does not exist. 
EMULATOR: SUCCESS -- Response: 



count: 4   ------------------------------------------------------------------------------------------------------------------------------------------------------------------------


DISCREPANT METHOD: s3_get_bucket_inventory_configuration --- ARGS: []
--Behavior mismatch--
CLOUD: FAILURE -- Response: An error occurred (NoSuchConfiguration) when calling the GetBucketInventoryConfiguration operation: The specified configuration does not exist. 
EMULATOR: SUCCESS -- Response: 



count: 5   ------------------------------------------------------------------------------------------------------------------------------------------------------------------------


DISCREPANT METHOD: s3_get_bucket_metrics_configuration --- ARGS: []
--Behavior mismatch--
CLOUD: FAILURE -- Response: An error occurred (NoSuchConfiguration) when calling the GetBucketMetricsConfiguration operation: The specified configuration does not exist. 
EMULATOR: SUCCESS -- Response: 



count: 6   ------------------------------------------------------------------------------------------------------------------------------------------------------------------------


DISCREPANT METHOD: s3_get_bucket_policy --- ARGS: []
--Behavior mismatch--
CLOUD: FAILURE -- Response: An error occurred (NoSuchBucketPolicy) when calling the GetBucketPolicy operation: The bucket policy does not exist 
EMULATOR: SUCCESS -- Response: 



count: 7   ------------------------------------------------------------------------------------------------------------------------------------------------------------------------


DISCREPANT METHOD: s3_get_bucket_policy_status --- ARGS: []
--Behavior mismatch--
CLOUD: FAILURE -- Response: An error occurred (NoSuchBucketPolicy) when calling the GetBucketPolicyStatus operation: The bucket policy does not exist 
EMULATOR: SUCCESS -- Response: 



count: 8   ------------------------------------------------------------------------------------------------------------------------------------------------------------------------


DISCREPANT METHOD: s3_get_bucket_replication --- ARGS: []
--Behavior mismatch--
CLOUD: FAILURE -- Response: An error occurred (ReplicationConfigurationNotFoundError) when calling the GetBucketReplication operation: The replication configuration was not found 
EMULATOR: SUCCESS -- Response: 



count: 9   ------------------------------------------------------------------------------------------------------------------------------------------------------------------------


DISCREPANT METHOD: s3_get_object_retention --- ARGS: []
--Behavior mismatch--
CLOUD: SUCCESS -- Response:  
EMULATOR: FAILURE -- Response: An error occurred (InternalError) when calling the GetObjectRetention operation (reached max retries: 4): exception while calling s3.GetObjectRetention: Unable to parse response (syntax error: line 1, column 0), invalid XML received. Further retries may succeed:
b'test'



count: 10   ------------------------------------------------------------------------------------------------------------------------------------------------------------------------


DISCREPANT METHOD: s3_get_object_torrent --- ARGS: []
--Behavior mismatch--
CLOUD: FAILURE -- Response: An error occurred (MethodNotAllowed) when calling the GetObjectTorrent operation: The specified method is not allowed against this resource. 
EMULATOR: SUCCESS -- Response: 



count: 11   ------------------------------------------------------------------------------------------------------------------------------------------------------------------------


DISCREPANT METHOD: s3_put_bucket_acl --- ARGS: []
--Behavior mismatch--
CLOUD: FAILURE -- Response: An error occurred (AccessDenied) when calling the PutBucketAcl operation: Access Denied 
EMULATOR: SUCCESS -- Response: 



count: 12   ------------------------------------------------------------------------------------------------------------------------------------------------------------------------


DISCREPANT METHOD: s3_put_bucket_analytics_configuration --- ARGS: []
--Behavior mismatch--
CLOUD: SUCCESS -- Response:  
EMULATOR: FAILURE -- Response: An error occurred (IllegalLocationConstraintException) when calling the PutBucketAnalyticsConfiguration operation: The unspecified location constraint is incompatible for the region specific endpoint this request was sent to.



count: 13   ------------------------------------------------------------------------------------------------------------------------------------------------------------------------


DISCREPANT METHOD: s3_put_bucket_intelligent_tiering_configuration --- ARGS: []
--Behavior mismatch--
CLOUD: SUCCESS -- Response:  
EMULATOR: FAILURE -- Response: An error occurred (IllegalLocationConstraintException) when calling the PutBucketIntelligentTieringConfiguration operation: The unspecified location constraint is incompatible for the region specific endpoint this request was sent to.



count: 14   ------------------------------------------------------------------------------------------------------------------------------------------------------------------------


DISCREPANT METHOD: s3_put_bucket_inventory_configuration --- ARGS: []
--Behavior mismatch--
CLOUD: SUCCESS -- Response:  
EMULATOR: FAILURE -- Response: An error occurred (IllegalLocationConstraintException) when calling the PutBucketInventoryConfiguration operation: The unspecified location constraint is incompatible for the region specific endpoint this request was sent to.



count: 15   ------------------------------------------------------------------------------------------------------------------------------------------------------------------------


DISCREPANT METHOD: s3_put_bucket_logging --- ARGS: []
--Behavior mismatch--
CLOUD: SUCCESS -- Response:  
EMULATOR: FAILURE -- Response: An error occurred (InvalidTargetBucketForLogging) when calling the PutBucketLogging operation: You must give the log-delivery group WRITE and READ_ACP permissions to the target bucket



count: 16   ------------------------------------------------------------------------------------------------------------------------------------------------------------------------


DISCREPANT METHOD: s3_put_bucket_metrics_configuration --- ARGS: []
--Behavior mismatch--
CLOUD: SUCCESS -- Response:  
EMULATOR: FAILURE -- Response: An error occurred (IllegalLocationConstraintException) when calling the PutBucketMetricsConfiguration operation: The unspecified location constraint is incompatible for the region specific endpoint this request was sent to.



count: 17   ------------------------------------------------------------------------------------------------------------------------------------------------------------------------


DISCREPANT METHOD: s3_put_bucket_policy --- ARGS: []
--Behavior mismatch--
CLOUD: FAILURE -- Response: An error occurred (AccessDenied) when calling the PutBucketPolicy operation: Access Denied 
EMULATOR: SUCCESS -- Response: 



count: 18   ------------------------------------------------------------------------------------------------------------------------------------------------------------------------


DISCREPANT METHOD: s3_put_bucket_replication --- ARGS: []
--Behavior mismatch--
CLOUD: FAILURE -- Response: An error occurred (AccessDenied) when calling the PutBucketReplication operation: Access Denied 
EMULATOR: SUCCESS -- Response: 



count: 19   ------------------------------------------------------------------------------------------------------------------------------------------------------------------------


DISCREPANT METHOD: s3_put_object_acl --- ARGS: []
--Behavior mismatch--
CLOUD: FAILURE -- Response: An error occurred (AccessDenied) when calling the PutObjectAcl operation: Access Denied 
EMULATOR: SUCCESS -- Response: 



count: 20   ------------------------------------------------------------------------------------------------------------------------------------------------------------------------


DISCREPANT METHOD: s3_put_object_retention --- ARGS: []
--Behavior mismatch--
CLOUD: FAILURE -- Response: An error occurred (AccessDenied) when calling the PutObjectRetention operation: Access Denied 
EMULATOR: SUCCESS -- Response: 



count: 21   ------------------------------------------------------------------------------------------------------------------------------------------------------------------------


DISCREPANT METHOD: s3_restore_object --- ARGS: []
--Status code mismatch--
CLOUD: FAILURE -- HTTP Status code: 404 -- Error Code: NoSuchKey -- Error Message: The specified key does not exist.
EMULATOR: FAILURE -- HTTP Status code: 500 -- Error Code: InternalError -- Error Message: exception while calling s3.RestoreObject: 'NoneType' object has no attribute 'storage_class'



count: 22   ------------------------------------------------------------------------------------------------------------------------------------------------------------------------


DISCREPANT METHOD: s3_select_object_content --- ARGS: []
--Behavior mismatch--
CLOUD: SUCCESS -- Response:  
EMULATOR: FAILURE -- Response: An error occurred (InternalError) when calling the SelectObjectContent operation (reached max retries: 4): exception while calling s3.SelectObjectContent: 'Variable' object is not iterable



count: 23   ------------------------------------------------------------------------------------------------------------------------------------------------------------------------


***  Round Summary  ***

Behavior mismatch count: 21
Status code mismatch count: 2
Error message mismatch count: 0
Total discrepancy count: 23/98


Round ended   ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------


